bokehjs/src/coffee/polyfill.ts

```diff
@@ -0,0 +1,9 @@
+declare function require(moduleName: string): any
+
+if (typeof WeakMap !== "function") {
+  require("es6-weak-map/implement")
+}
+
+if (typeof Set !== "function") {
+  require("es6-set/implement")
+}
```

Comment:
This is annoying, but phantomjs and IE 10 don't support this. phantomjs isn't going to be maintained anymore, so we need to switch to chrome/chromium headless sooner than later. I'm not sure if we support IE 10, but at least as long as we use phantomjs, it's convenient though expensive to have this on-board. It's expensive, because it adds some 40~50 kB minified. When we drop phantomjs, then for IE we can provide a separate file with polyfills, so that users of the default, modern browsers don't have to suffer bigger downloads.